pandas/tools/plotting.py

```diff
@@ -1533,8 +1558,11 @@ class PlanePlot(MPLPlot):
 
     _layout_type = 'single'
 
-    def __init__(self, data, x, y, **kwargs):
-        MPLPlot.__init__(self, data, **kwargs)
+    def __init__(self, data, x, y, sharex=False, **kwargs):
+        if sharex is None:
```

Comment:
To apply the same fix to hexbin, pls move the logic to ``PlanePlot``. And I _think_ ``sharex`` should be always ``False`` because we don't support scatter subplots ATM?